apps/video/test/marionette/lib/video_options_view.js

```diff
@@ -0,0 +1,89 @@
+/* global module */
+(function(module) {
+  'use strict';
+
+  /**
+   * @constructor
+   * @param {Marionette.Client} client Marionette client to use.
+   */
+  function VideoOptionsView(client) {
+    this.client = client.scope({
+    	searchTimeout: 20000
+  	});
+  }
+
+  module.exports = VideoOptionsView;
+
+  /**
+   * @const {Object}
+   */
+  VideoOptionsView.Selector = Object.freeze({
+    optionsView: '#options-view',
+    shareButton: '.single-share-button',
+    infoButton: '.single-info-button',
+    deleteButton: '.single-delete-button',
+    cancelButton: '.gaia-menu-cancel'
+  });
+
+  VideoOptionsView.prototype = {  
+    
+    client: null,
+
+    /**
+     * @return {boolean} the display status of the options view
+     */
+    get optionsViewDisplayed() {
+      return this.client.helper.waitForElement(
+      VideoOptionsView.Selector.optionsView).displayed();
+    },
+
+    /**
+    * @return {Marionette.element} share button object
+    */
+  	get shareButton() {
+      return this.client.helper.waitForElement(
+        VideoOptionsView.Selector.shareButton);
+    },
+
+    /**
+    *  @return {Marionette.element} more info button object
+    */
+    get infoButton() {
+      return this.client.helper.waitForElement(
+        VideoOptionsView.Selector.infoButton);
+    },
+
+    /**
+    *  @return {Marionette.element} delete button object
+    */
+    get deleteButton() {
+      return this.client.helper.waitForElement(
+        VideoOptionsView.Selector.deleteButton);
+    },
+
+    /**
+    *  @return {Marionette.element} cancel button object
+    */
+    get cancelButton() {
+      return this.client.helper.waitForElement(
+        VideoOptionsView.Selector.cancelButton);
+    },
+
+    tapShare: function() {
+      this.shareButton.click();
+    },
+
+    tapMoreInfo: function() {
```

Comment:
Do you think this function should wait for the info view to be displayed? It would prevent timing problems where the caller calls 'tapMoreInfo' and then tests for an info view property before the info view has been displayed.